DSP_Plugins.GalacticScale/Scripts/PatchStarSystemGeneration/Rework/ReworkStarGen.cs

```diff
@@ -315,7 +319,7 @@ public static class ReworkStarGen {
                             currentOrbitMoonIndex += UnityRandom.Range(currentOrbitMoonIndex, currentOrbitMoonIndex + currentSettings.JumpOrbitMoonMax);
                         }
 
-                    currentPlanet.AddMoonInOrbit(nbOfBodiesPreGenerated, currentOrbitMoonIndex, genSeed, infoSeed);
+                    currentPlanet.AddMoonInOrbit(nbOfBodiesPreGenerated, currentOrbitMoonIndex, planetGenSeed, planetInfoSeed);
```

Comment:
Is it intentional that genSeed and infoSeed are reversed here, compared to the order in the Add call? They don't _need_ to match but it feels like it might be better to sync the two signatures up.